CRM/Case/Info.php

```diff
@@ -245,4 +245,40 @@ public static function onToggleComponents($oldValue, $newValue, $metadata) {
     }
   }
 
+  /**
+   * @return array
+   *   Array(string $value => string $label).
+   */
+  public static function getRedactOptions() {
+    return array(
+      'auto' => ts('Auto'),
```

Comment:
Do you think "Auto" will make sense to a user here? I've seen the help text in the template and `settings/Case.setting.php` file. I'm not so familiar with the context, but just thought something like "default" would be more standard here.